src/Compilers/CSharp/Portable/CSharpCodeAnalysis.csproj

```diff
@@ -832,13 +59,7 @@
     <BoundTreeDefinition Include="BoundTree\BoundNodes.xml">
       <SubType>Designer</SubType>
     </BoundTreeDefinition>
-    <Compile Include="Generated\BoundNodes.xml.Generated.cs" />
```

Comment:
@jasonmalinowski These are no longer automatically generated during the build process, but rather on demand by running a script.